e2e/pvc.go

```diff
@@ -61,14 +61,14 @@ func createPVCAndvalidatePV(c kubernetes.Interface, pvc *v1.PersistentVolumeClai
 
 		pv, err = c.CoreV1().PersistentVolumes().Get(context.TODO(), pvc.Spec.VolumeName, metav1.GetOptions{})
 		if err != nil {
+			if isRetryableAPIError(err) {
```

Comment:
[e2e run](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.19/detail/mini-e2e-helm_k8s-1.19/1999/pipeline/) ([logs](https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/rest/organizations/jenkins/pipelines/mini-e2e-helm_k8s-1.19/runs/1999/nodes/98/log/?start=0))
```
Jul 13 18:21:12.215: INFO: Error getting pvc in namespace: '': etcdserver: request timed out
Jul 13 18:21:12.981: INFO: waiting for PVC  (102 seconds elapsed)
Jul 13 18:21:12.981: INFO: Error getting pvc in namespace: '': an empty namespace may not be set when a resource name is provided
Jul 13 18:21:12.981: INFO: failed to create PVC (rbd-32740): failed to get pvc: an empty namespace may not be set when a resource name is provided
```
@nixpanic Looks like we are overwriting on existing `pvc` variable with a empty `&PVC{}` if there's a error during `Get` at line 46, probably need to use another variable to avoid this.

Reply:
Indeed! That is really ugly :disappointed:

Reply:
now using an extra `namespace` variable to store the value that is initially used (just like `name` refers to `pvc.Name`).